src/Element/MineOrganisationsData.php

```diff
@@ -152,6 +152,7 @@ public static function getCompositeElements(array $element): array {
       '#type' => 'select',
       '#title' => t('Organisations funktion'),
       '#options' => [],
+      '#access' => FALSE,
```

Comment:
Should we use a setting for disabling `organisations_funktion`?

Reply:
Added comment explaining that it is handled later in 
`\Drupal\os2forms_organisation\Plugin\WebformElement\MineOrganisationsData::alterForm`.